scripts/process-image-manifests

```diff
@@ -74,20 +74,34 @@ get_current_csv() {
     "3scale")
        CURRENT_CSV=$(yq r ${MANIFESTS_DIR}/integreatly-${PRODUCT}/${CURRENT_CSV_FILE_NAME} channels[0].currentCSV | sed -e s/^3scale-operator.v//)
        ;;
+    "amq-online")
+       CURRENT_CSV=$(yq r ${MANIFESTS_DIR}/integreatly-${PRODUCT}/${CURRENT_CSV_FILE_NAME} channels[0].currentCSV | sed -e s/^amqonline.//)
+       ;;
      *)
        echo "Passed an unknown product $PRODUCT"
        exit
        ;;
     esac
     echo "Current CSV $CURRENT_CSV"
-
 }
```

Comment:
This whole thing feels like something we should be trying to extract into the csv_helper module that we have here https://github.com/integr8ly/integreatly-operator/blob/delorean-ews/scripts/csv_helper

We should be able to use this in some way https://github.com/integr8ly/integreatly-operator/blob/delorean-ews/scripts/csv_helper#L10, and ideally make it not product specific i.e. extract everything off the string apart from the version bit you are looking for here.